packages/core/core/src/ResolverRunner.js

```diff
@@ -33,8 +35,49 @@ export default class ResolverRunner {
 
     let resolvers = await this.config.getResolvers();
 
+    let pipeline;
+    let filePath;
+    if (dependency.moduleSpecifier.includes(':')) {
+      [pipeline, filePath] = dependency.moduleSpecifier.split(':');
+      let transformsWithPipelines = {};
+      for (let key of Object.keys(this.config.transforms)) {
```

Comment:
Should we also include optimizers here? What if you had a named pipeline that only existed in the optimizer config?

Reply:
Good point. Think we should be using the config matching method or just extract the pipelines ourselves here and match against them?

Reply:
Perhaps we could extract the named pipelines once and cache them? Maybe in the ParcelConfig class?